x-pack/plugins/ml/public/application/explorer/components/explorer_no_jobs_found/explorer_no_jobs_found.js

```diff
@@ -12,20 +12,36 @@ import React from 'react';
 import { FormattedMessage } from '@kbn/i18n/react';
 
 import { EuiEmptyPrompt, EuiButton } from '@elastic/eui';
-
-export const ExplorerNoJobsFound = () => (
-  <EuiEmptyPrompt
-    iconType="alert"
-    title={
-      <h2>
-        <FormattedMessage id="xpack.ml.explorer.noJobsFoundLabel" defaultMessage="No jobs found" />
-      </h2>
-    }
-    actions={
-      <EuiButton color="primary" fill href="ml#/jobs">
-        <FormattedMessage id="xpack.ml.explorer.createNewJobLinkText" defaultMessage="Create job" />
-      </EuiButton>
-    }
-    data-test-subj="mlNoJobsFound"
-  />
-);
+import { ML_PAGES } from '../../../../../common/constants/ml_url_generator';
+import { Link } from 'react-router-dom';
```

Comment:
Nit: This import could be at the top of the file.

Reply:
Updated here cc8f5dd